### ISSUE TITLE
PeNet attempts to parse NE file as PE file

### ISSUE THREAD

**Issue by rweijnen** [secana/PeNet#329]
When PeNet reads a NE (new executable) file (16-bit) it attemps to parse it as a PE file, sometimes it hangs and sometimes it reports incorrect information.

Here is an example file:
[COMMDLG.ZIP](https://github.com/secana/PeNet/files/15321269/COMMDLG.ZIP)

with below code, IsDotNet reports true for example (tested with v4.0.4):

```
        static void Main(string[] args)
        {
            string filename = "COMMDLG.DLL";
            PeFile peFile = new PeFile(filename);
            Console.WriteLine($"Is DLL: {peFile.IsDll}");
            Console.WriteLine($"Is EXE: {peFile.IsExe}");
            Console.WriteLine($"Is DRV: {peFile.IsDriver}");
            Console.WriteLine($"Is NET: {peFile.IsDotNet}");
            Console.WriteLine($"32-bit: {peFile.Is32Bit}");
            Console.WriteLine($"64-bit: {peFile.Is64Bit}");
        }

```

Expected behaviour: fail to load file or produce exception.

**Comment by secana** [comment]
Thanks for reporting. I'll have a look.

**Comment by secana** [comment]
Fixed in 4.0.5
PeNet throws an error now that the file is not a valid PE file.
